docs/UserGuide.md

```diff
@@ -4,339 +4,1044 @@
+
+#### Example:
+
+Suppose you just had an interview for the third internship application on your list, and you'd like to add a note 
+mentioning that the **interviewer emphasized a follow-up in two weeks**. The command would look like this:
+
+```
+note 3 i/Interviewer emphasized follow-up in two weeks.
+```
+
+**_UPDATE PICTURE HERE_**
 
-**Expected Output**:
 ![note command example](images/noteExample.jpg)
 
-### Deleting an internship application entry : `delete`
+You can see how the note gets attached to the specified internship application. 
+This ensures you have a comprehensive view of each application, complete with any supplementary details or reminders you've deemed important.
+
+<box type="tip" light>
+
+* Making a habit of attaching notes, especially after interviews or significant interactions, will ensure you don't forget crucial details.
+* Reviewing notes can be particularly helpful when preparing for subsequent rounds of interviews or follow-ups.
+
+</box>
+
+__________________________________________________________________________________________________________________
+
+### Deleting an internship application : `delete`
+
+The `delete` command is an essential tool in your internship application management toolkit, designed to help you **maintain a clean and relevant list** of applications. 
+
+As situations change, such as when you decide not to pursue a particular opportunity or when an application is no longer relevant, 
+the `delete` command ensures you can easily remove it from your records.
+
+#### Command Usage
+
+**Command**:
+```
+delete INDEX
+```
+
+#### Parameters:
+
+1. **INDEX**: Represents the position of the internship application in your list that you wish to delete.
+   <box type="info" seamless>
 
-Deletes the specified internship application entry from the internship list.
+   The `INDEX` should be a positive integer. For example: `1`, `2`, `3`, etc. This corresponds to the application's position in your list.
+   </box>
 
-**Format**: `delete INDEX`
+#### Example:
 
-**Parameters**:
-* `INDEX`: Index of internship application entry to be deleted.
-    * `INDEX` must be a positive integer, such as, `1`, `2`, `3`, etc.
+If you decide that the fourth internship application on your list is no longer relevant, and you want to remove it, the command would look like this:
 
-Examples:
-* `delete 1` deletes the first internship application in the list.
+```
+delete 4
+```
+
+**_UPDATE PICTURE HERE_**
 
-**Expected Output**: 
 ![delete command example](images/deleteExample.jpg)
 
+You can see how the specified internship application is removed from the list. 
+The remaining applications adjust their index numbers accordingly, ensuring a continuous sequence.
+
+<box type="warning" light>
+
+* Before executing the `delete` command, always double-check the `INDEX` to ensure you are removing the correct application.
```

Comment:
It is no longer irreversible bc u can undo

Reply:
Okay! Thanks :D